docs/pages/project/contributing/ci-build-and-release.md

```diff
@@ -1,10 +1,11 @@
 ---
-layout: default
```

Comment:
What's driving the need for this layout change? Are the other contributing pages using `page` as their layout, too?

Reply:
Yes, the other contributing pages using `page` as their layout, so I changed it.

Reply:
Gotcha. Thank you, @sambit-sankalp 👍